crates/hc/Cargo.toml

```diff
@@ -37,5 +37,9 @@ tokio = { version = "1.36.0", features = ["full"] }
 workspace = true
 
 [features]
+default = ["wasmer_sys"]
 
 chc = ["holochain_cli_sandbox/chc"]
+
+wasmer_sys = ["holochain_cli_bundle/wasmer_sys"]
```

Comment:
Can we make it so that `hc-bundle` always supports both? Or can they not exist side by side? 

It would be a pain to have to include multiple builds of `hc` in a Holonix environment to bundle for different platforms.

Reply:
I guess this would be blocked by building for Windows actually wouldn't it? 

I'd probably prefer a platform exclude. Like a dependency on wasmer-wamr if not on platform windows. Then we can quietly enable Windows support later when we figure out building for it

Reply:
Removing the feature flag entirely there's no need for it in hc.